### PR TITLE
No-cache headers on done()

### PR DESCRIPTION
**Pull request by n321203**
Internet Explorer needs to see no-cache headers, otherwise it will cache the last API response and always return that (ignoring any changes). 

In the current master branch, no-cache headers are not set for PUT:s, which means if you update a collections multiple times (using PUT), IE wont recognize the changes. 

This patch adds no-cache headers to all API responses in the done() function of context.js, so that IE behaves nicely again...

### CODE REVIEW

**Comment by NicolasRitouet** [comment]
Hi @n321203 
Thanks for your PR.
However, it's better for us to have two separate for the two features.
It allows us to:
- discuss separately about the features
- have a cleaner changelog
- have easier eventual rollback
Could you please separate them?

**Comment by n321203** [comment]
Done – this PR now only adds no-cache headers

**Comment by andreialecu** [comment]
+1, seems innocent enough.

File: lib/context.js [hunk 1]
```diff
@@ -92,6 +92,9 @@ Context.prototype.done = function(err, res) {
     }
 
     try {
+      this.res.setHeader("Cache-Control", "no-cache, no-store, must-revalidate");
+      this.res.setHeader("Pragma", "no-cache");
+      this.res.setHeader("Expires", "0");
```
Comment (ericfong) [inline lib/context.js]: Shall we use ```this.res. getHeader``` to check a value have been set or not?

That can increase the flexibility of dpd-module / dpd-plugin

Comment (NicolasRitouet) [inline lib/context.js]: We can add it later in another PR if some people request it.

**Comment by ericfong** [comment]
Thanks for your contribution! @n321203 
Landed as 067e72ac96a15f4b057d058bdc84f5feb5333351

**Pull request closed without merging**